### PR TITLE
Create Collective Code Ownership wiki

### PR DESCRIPTION
**Pull request by mariamcquillan**

### CODE REVIEW

File: practices/collectiveCode/readme.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+# Collective Code Ownership
+
+In collective code ownership, the entire team is responsible for the code. Everyone works together to produce a product of quality. No one individual is greater than the rest of the team members. This practice helps to reduce the need for hero work and eliminate silos of information within the team.
+
+## The What and the Why
+
+### The What
+
+Code ownership refers to who is allowed to modify and write certain portions of the code. Collective code ownership means the whole team has the ability and even duty to review and edit every piece of code.
```
Comment (headl033) [inline practices/collectiveCode/readme.md:9]: 'review' has some baggage in this context that doesn't really apply, 'understand' might be a better fit

File: practices/collectiveCode/readme.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+### The Why
+
+Benefits of collective code ownership:
+
+1. Increases work velocity by not requiring work from a specific developer.
```
Comment (headl033) [inline practices/collectiveCode/readme.md:15]: list should probably be unordered

File: practices/collectiveCode/readme.md (hunk 1 above)
Comment (mariamcquillan) [reply practices/collectiveCode/readme.md:9]: fixed!

File: practices/collectiveCode/readme.md (hunk 2 above)
Comment (mariamcquillan) [reply practices/collectiveCode/readme.md:15]: updated to bulleted list

**Review by headl033 — APPROVED** [review]

**Pull request merged**